lib/l10n/app_de.arb

```diff
@@ -186,10 +186,12 @@
     "settingsLockWallet": "Wallet sperren",
     "settingsHeaderSecurity": "Sicherheit",
     "settingsLabelChangePassword": "Passwort ändern",
```

Comment:
Probably translators apps propose a good translation for Privacy Policy, but in general, unless that you have a really good knowledge on the language, please do not translate the strings. 
When preparing the release, I'll request translation to people who is native/profit in each language.